WoWDatabaseEditor.Common/WDE.CMMySqlDatabase/Models/QuestTemplateAddon.cs

```diff
@@ -0,0 +1,18 @@
+﻿using LinqToDB.Mapping;
+using WDE.Common.Database;
+
+namespace WDE.CMMySqlDatabase.Models
+{
```

Comment:
Let's just remove the file if there is no such table.

Reply:
removed 👍